### ISSUE TITLE
react/jsx-indent: Closing parenthese indentation

### ISSUE THREAD

**Issue by stefanbuck** [yannickcr/eslint-plugin-react#618]
The indentation of the closing parentheses on return statements seems to be not working.

The following patterns are considered warnings:

``` js
return (
  <div />
  );
```

``` js
return (
  <div />);
```

The following patterns are not considered warnings:

``` js
return (
  <div />
);
```

``` js
return (<div />);
```

**Comment by ljharb** [comment]
The examples you've mentioned are exactly how it should be working. Is it working differently?

**Comment by stefanbuck** [comment]
Mhhh, good to know. So it must be a configuration issue, right? I tried to find the conflicting rule without success. Would it be possible that you take a look?

https://github.com/stefanbuck/eslint-plugin-react-issues-618

**Comment by ljharb** [comment]
@stefanbuck thanks for the repro repo! It definitely looks like a bug.

**Comment by stefanbuck** [comment]
Is [jsxp-indent.js](https://github.com/yannickcr/eslint-plugin-react/blob/master/lib/rules/jsx-indent.js) the right place to fix it? I'll try to come up with a PR for this.
